src/qt/peertablesortproxy.cpp

```diff
@@ -38,6 +38,8 @@ bool PeerTableSortProxy::lessThan(const QModelIndex& left_index, const QModelInd
         return left_stats.nRecvBytes < right_stats.nRecvBytes;
     case PeerTableModel::Subversion:
         return left_stats.cleanSubVer.compare(right_stats.cleanSubVer) < 0;
+    case PeerTableModel::Bumper:
+        return left_stats.nodeid < right_stats.nodeid;
```

Comment:
Why this sorting scheme?

Reply:
Sorting by ```nodeid``` was simply to have sorting functionality for that column as opposed to no sorting functionality.